index.bs

```diff
@@ -1167,6 +1167,17 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
             when the [=public key credential source=] was [=registration|registered=].
             Storing this in combination with the above [$credential record/attestationObject$] [=struct/item=]
             enables the [=[RP]=] to re-verify the [=attestation signature=] at a later time.
+
+        :   <dfn>displayName</dfn>
```

Comment:
I think there is possible confusion here because displayName generally refers to the visible name of the account holder.  I think this needs to be named something more explicit to make it completely clear exactly what it refers to because otherwise RP's may confuse this and end up with dialogs that say "User your credential for site X from Password Manager Steve Bobington".

Reply:
Yes, my bad - I renamed `{{CredentialPropertiesOutput/displayName}}` but forgot to also rename `[$credential record/displayName$]`. Both are renamed to `authenticatorDisplayName` now.

Reply:
Thank you!